docs/workflow/requirements/linux-requirements.md

```diff
@@ -26,6 +26,13 @@ Minimum RAM required to build is 1GB. The build is known to fail on 512 MB VMs (
 
 These instructions are written assuming the current Ubuntu LTS.
 
+You can use this helper script to install dependencies on some platforms:
```

Comment:
Should this be mentioned before "Debian-based / Ubuntu"? It applies to all distros.